src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,28 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
```

Comment:
This class is not called by any other classes. According to the diagram, this class should be instantiated and kept track of by AddressBook.